paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PrepareResourcesTask.kt

```diff
@@ -21,36 +21,36 @@ import com.android.Version
 import com.android.ide.common.symbols.getPackageNameFromManifest
 import org.gradle.api.DefaultTask
 import org.gradle.api.Project
-import org.gradle.api.file.Directory
 import org.gradle.api.file.DirectoryProperty
+import org.gradle.api.file.RegularFile
 import org.gradle.api.provider.Provider
-import org.gradle.api.tasks.Internal
+import org.gradle.api.tasks.Input
+import org.gradle.api.tasks.OutputFile
 import org.gradle.api.tasks.TaskAction
 import org.gradle.api.tasks.TaskProvider
 import org.gradle.util.VersionNumber
 import java.io.File
 
 open class PrepareResourcesTask : DefaultTask() {
-  // Replace with @InputDirectory once mergeResourcesProvider.outputDir is of type Provider<File>.
 
   internal lateinit var mergeResourcesProvider: TaskProvider<MergeResources>
-    @Internal get
+    @Input get
```

Comment:
Marking this as input for caching and up-to-date (I did not add [CacheableTask](https://docs.gradle.org/current/javadoc/org/gradle/api/tasks/CacheableTask.html) annotation, though. I wasn't sure if this is something I wanted to do)